serialize.h

```diff
@@ -60,7 +60,7 @@ class CDataStream;
 class CAutoFile;
 static const unsigned int MAX_SIZE = 0x02000000;
 
-static const int PROTOCOL_VERSION = 170013;
+static const int PROTOCOL_VERSION = 170014;
```

Comment:
This is the correct version to update to ensure that the testnet seeders will still be able to connect to peers after NU5 testnet activation.